content/projects/gsoc/2023/project-ideas/gitlab-plugin-modernization.adoc

```diff
@@ -11,7 +11,7 @@ skills:
 - Docker
 - GitLab
 mentors:
-- "gsoc"
+- "basil"
```

Comment:
The addition of my name as a mentor for this project implies my endorsement of the content of this page, but I was not consulted before this page was added, and in fact some parts of it (not written by me) seem suspect as described in https://github.com/jenkins-infra/jenkins.io/pull/5836#discussion_r1087096858.

> This PR updates GSoC project ideas with the names of those that volunteered or mentioned an interest in mentoring a project

This is not accurate. In the private discussion in which I volunteered to help mentor this project, I specifically volunteered to mentor only the RESTeasy portion. One way to resolve https://github.com/jenkins-infra/jenkins.io/pull/5836#discussion_r1087096858 would be to retain this as a 175-hour project and limit the scope to just RESTeasy, in which case my assignment as the lead mentor would be consistent with what I volunteered for. But the current definition of the project as a 175-hour project with 3 or 4 major portions is consistent with neither my intended project size nor what I signed up to mentor.

Reply:
Being mentioned does not imply to be (lead) mentor of the project. At this stage, it only implies that this person will be in a position to answer the questions and guide the students during the redaction phase.

I apologize if I misunderstood the information that you gave me. 
To try to move this forward, we have the following options:
* revert this part of the draft PR (removing your name from it)
* remove the project idea completely
* or help me to rewrite the proposal in a more adequate way

Reply:
> Being mentioned does not imply to be (lead) mentor of the project.

Thanks! What I am trying to determine is whether the stakeholder for each project, id est, the person who will be responsible for maintaining the end result (if the project is not a proof of concept), has signed off on each project's description, size, and deliverables. (I had not for at least two projects.)

> To try to move this forward, we have the following options:

Any of these options are fine with me. I provided more detail about the third option in https://github.com/jenkins-infra/jenkins.io/pull/5836#issuecomment-1404202964.